ivy_tests/test_ivy/test_frontends/test_numpy/test_indexing_routines/test_indexing_like_operations.py

```diff
@@ -178,3 +178,45 @@ def test_numpy_compress(
         a=arr[0],
         axis=ax,
     )
+
+
+# take
+@handle_frontend_test(
+    fn_tree="numpy.take",
+    dtype_x_indices_axis=helpers.array_indices_axis(
+        array_dtypes= helpers.get_dtypes("numeric"),
+        indices_dtypes=["int32", "int64"],
+        min_num_dims=1,
+        max_num_dims=5,
+        min_dim_size=1,
+        max_dim_size=10,
+        indices_same_dims=True,
+        valid_bounds=False,
+    ),
+    mode=st.sampled_from(["clip", "wrap", "raise"]),
+    test_with_out=st.just(False)
+)
+def test_numpy_take(
```

Comment:
Seems to be an issue with arguments according to tests [here](https://github.com/unifyai/ivy/actions/runs/5659074707/job/15331825717?pr=19725#step:3:182). Would double check that all of your args and kwargs are correct :)